### PR TITLE
CoroutineScope job should be canceled before destory

### PR DESCRIPTION
**Pull request by daichen**
tombstone happens if run destory while loadModelGltfAsync in ModelViewer.kt

To Reproduce
1.loadModelGltfAsync
2.detach from the winodw quickly
3.tombstone ..

### CODE REVIEW

File: android/filament-utils-android/src/main/java/com/google/android/filament/utils/ModelViewer.kt [hunk 1]
```diff
@@ -269,6 +272,7 @@ class ModelViewer(val engine: Engine) : android.view.View.OnTouchListener {
         view.addOnAttachStateChangeListener(object : android.view.View.OnAttachStateChangeListener {
             override fun onViewAttachedToWindow(v: android.view.View?) {}
             override fun onViewDetachedFromWindow(v: android.view.View?) {
+                fetchResJob?.cancel()
```
Comment (romainguy) [inline android/filament-utils-android/src/main/java/com/google/android/filament/utils/ModelViewer.kt:275]: destroyModel() is called below which will call cancel on the job, is it necessary to cancel here as well?

File: android/filament-utils-android/src/main/java/com/google/android/filament/utils/ModelViewer.kt [hunk 2]
```diff
@@ -85,6 +85,8 @@ class ModelViewer(val engine: Engine) : android.view.View.OnTouchListener {
     private var surfaceView: SurfaceView? = null
     private var textureView: TextureView? = null
 
+    private var fetchResJob: Job? = null
```
Comment (romainguy) [inline android/filament-utils-android/src/main/java/com/google/android/filament/utils/ModelViewer.kt:88]: Please rename to `fetchResourcesJob`

**Review by romainguy — CHANGES_REQUESTED** [review]
Thank you for your contribution! I only have a couple of comments

**Comment by daichen** [comment]
sorry, I'm not familiar with git hub, I will close this request and create a new request

**Pull request closed without merging**